src/logging.cpp

```diff
@@ -91,7 +110,7 @@ init(bool console, quill::LogLevel log_level, bool structured)
 #else
     if (structured) {
         file_handler = quill::create_handler<quill::JsonFileHandler>(
-            (LOG_FILE + ".json").string(),
+            (LOG_FILE / ".json").string(),
```

Comment:
```suggestion
            (LOG_FILE + ".json").string(),
```
NO. `/` adds a directory seperator.

Reply:
you wrote this my guy

Reply:
Did not.